torch/testing/_comparison.py

```diff
@@ -875,8 +911,24 @@ def originate_pairs(
 
     else:
         for pair_type in pair_types:
-            with contextlib.suppress(UnsupportedInputs):
+            try:
                 return [pair_type(actual, expected, id=id, **options)]
+            except UnsupportedInputs:
+                continue
```

Comment:
This series of excepts is going to need a comment (and comments for each except would be helpful, too!)